pkg/apis/sources/v1alpha1/register_test.go

```diff
@@ -22,12 +22,12 @@ import (
 )
 
 func TestRegisterHelpers(t *testing.T) {
-	if got, want := Kind("CouchDbSource"), "CouchDbSource.sources.knative.dev"; got.String() != want {
-		t.Errorf("Kind(CouchDbSource) = %v, want %v", got.String(), want)
+	if got, want := Kind("MongoDbSource"), "MongoDbSource.sources.knative.dev"; got.String() != want {
```

Comment:
this should break once you change the import. I'd suggest you change the import, see that it fails, and then fix it.

Reply:
Unsure

Reply:
Do unit test